dashboard/app/models/course_offering.rb

```diff
@@ -455,6 +455,14 @@ def hoc?
     marketing_initiative == Curriculum::SharedCourseConstants::COURSE_OFFERING_MARKETING_INITIATIVES.hoc
   end
 
+  def hoai?
+    marketing_initiative == Curriculum::SharedCourseConstants::COURSE_OFFERING_MARKETING_INITIATIVES.hoai
+  end
+
+  def hoc_or_hoai?
```

Comment:
if there is a better name for this method, I'm open to suggestions

Reply:
I think this sounds good!